tests/waku_archive/test_driver_postgres_query.nim

```diff
@@ -645,6 +645,50 @@ suite "Postgres driver - queries":
     check:
       filteredMessages == expected[2 .. 3].reversed()
 
+  asyncTest "only cursor - invalid":
+    ## Given
+    const contentTopic = "test-content-topic"
+
+    let expected =
+      @[
+        fakeWakuMessage(@[byte 0], ts = ts(00)),
+        fakeWakuMessage(@[byte 1], ts = ts(10)),
+        fakeWakuMessage(@[byte 2], contentTopic = contentTopic, ts = ts(20)),
+        fakeWakuMessage(@[byte 3], contentTopic = contentTopic, ts = ts(30)),
+        fakeWakuMessage(@[byte 4], contentTopic = contentTopic, ts = ts(40)),
+        fakeWakuMessage(@[byte 5], contentTopic = contentTopic, ts = ts(50)),
+        fakeWakuMessage(@[byte 6], contentTopic = contentTopic, ts = ts(60)),
+        fakeWakuMessage(@[byte 7], contentTopic = contentTopic, ts = ts(70)),
+      ]
+    var messages = expected
```

Comment:
We don't really use the `expected` variable here, so we can get rid of it (same in the other tests)

```suggestion
    let messages =
      @[
        fakeWakuMessage(@[byte 0], ts = ts(00)),
        fakeWakuMessage(@[byte 1], ts = ts(10)),
        fakeWakuMessage(@[byte 2], contentTopic = contentTopic, ts = ts(20)),
        fakeWakuMessage(@[byte 3], contentTopic = contentTopic, ts = ts(30)),
        fakeWakuMessage(@[byte 4], contentTopic = contentTopic, ts = ts(40)),
        fakeWakuMessage(@[byte 5], contentTopic = contentTopic, ts = ts(50)),
        fakeWakuMessage(@[byte 6], contentTopic = contentTopic, ts = ts(60)),
        fakeWakuMessage(@[byte 7], contentTopic = contentTopic, ts = ts(70)),
      ]
```

Reply:
yes! sorry for the copy pasta